api/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/hashicorp/hcl v1.0.0
 	github.com/hashicorp/vault/sdk v0.1.14-0.20190805214312-16112a336457
 	github.com/mitchellh/mapstructure v1.1.2
+	github.com/shirou/gopsutil v0.0.0-20190731134726-d80c43f9c984 // indirect
```

Comment:
You might wanna try `go mod tidy` to remove that line. Ignore it if you already did that 😄

Reply:
This is a direct dependency that got added as part of this PR (`go mod tidy` no-op'ed on `go.mod`).